src/Oden/Pretty.hs

```diff
@@ -124,7 +125,7 @@ instance Pretty Mono.Type where
   pp (Mono.TCon _ d r) = pp d <> parens (pp r)
   pp (Mono.TNoArgFn _ t) = rArr <+> pp t
   pp (Mono.TFn _ tf ta) = pp tf <+> rArr <+> pp ta
-  pp (Mono.TUncurriedFn _ as r) = hsep (punctuate (text "&") (map pp as)) <+> rArr <+> pp r
+  pp (Mono.TUncurriedFn _ as rs) = hsep (punctuate (text "&") (map pp as)) <+> rArr <+> hsep (punctuate (text "&") (map pp rs))
```

Comment:
Could we print the return types as `(a, b, c ...)`? The Oden user can see it as a tuple.